virttest/virt_vm.py

```diff
@@ -1002,7 +1012,10 @@ def verify_kernel_crash(self):
                 return
             match = re.search(panic_re, data, re.DOTALL | re.MULTILINE | re.I)
             if match:
-                raise VMDeadKernelCrashError(match.group(0))
+                if "BUG" in match.group(0):
```

Comment:
I don't think that we can judge the VM as dead according to ` if "BUG" in match.group(0)`.
I think we check the status of the VM and then raise the corresponding error.

Reply:
Thanks for the reminder, it has been corrected.